docs/adr/0002-videos-languages.md

```diff
@@ -39,19 +39,19 @@ We decided to take all these elements into account right from the
 beginning.
 
 So we have a main Django model named `Video`, from an author, with the
-link to to main video file, including the default audio track.
+link to the main video file, including the default audio track.
 
-For the other audio tracks, we have `AudioTrack` Django model, with a
+For the other audio tracks, we have an `AudioTrack` Django model, with a
 `ForeignKey` to the `Video` instance, named `video`, and a `language`
 field (with only one audio track for each video+language)
 
-It's the same for the subtitles, we have a `SubtitleTrack` Django model,
-with the same `video` and `language` fields, but with an additional `cc`
-field to indicate that this subtitle track is "[closed
-captioning](https://en.wikipedia.org/wiki/Closed_captioning)", ie a
-subtitles track for deaf or hard of hearing viewers. So there can be two
-subtitle tracks for each video+language: one with `cc` on, one with `cc`
-off.
+It's the same for timed text tracks, we have a `TimedTextTrack` Django model,
+with the same `video` and `language` fields, but with an additional `mode`
+field to indicate that this timed text track is either a simple subtitle,
+a "[closed
+captioning](https://en.wikipedia.org/wiki/Closed_captioning)" ie subtitles
+for deaf or hard of hearing viewers, or a transcript. So there can be up to 
+3 timed text tracks for each video+language: one in  each mode.
```

Comment:
you have two whitespaces `s/one in  each mode/one in each mode/` and maybe `one for each mode` is better ?